css-canvas-medical/styles.css

```diff
@@ -0,0 +1,254 @@
+.container {
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  flex-basis: 100%;
+  background-color: rgb(250 250 250);
+  padding-top: 5rem;
+  margin-top: 5rem;
```

Comment:
All of these should be moved to another class that you can add to the element in addition to the `column-full` class. `column-` classes should only have `flex-basis` or `width`, and _nothing else_